pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, lib, platforms,
+  mariadb, mariadb-connector-c,
+  boost, cmake, bison, icu, flex, }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, lib, platforms
, mariadb, mariadb-connector-c,
, boost, cmake, bison, icu, flex
}:
```

Reply:
Would you rather format like this, or shall I just run nixpkgs-fmt on it? Not sure if that's officially endorsed yet, but FWIW that results in:


```nix
{ stdenv
, fetchFromGitHub
, lib
, platforms
, mariadb
, mariadb-connector-c
, boost
, cmake
, bison
, icu
, flex
,
}:

let
  columnar = fetchFromGitHub {
    owner = "manticoresoftware";
    repo = "columnar";
    rev = "b211815f";
    sha256 = "sha256-7oWqa1P1sGUkaTuMLWz36KhR42SjEWQo/4g+JKAsJOU=";
  };
in


```